.eslintrc

```diff
@@ -58,9 +59,13 @@
 		"@typescript-eslint/explicit-module-boundary-types": "off",
 		"@typescript-eslint/no-require-imports": "error",
 		"@typescript-eslint/no-unused-expressions": "warn",
+		"curly": ["warn", "multi-line", "consistent"],
 		"prefer-const": "warn",
+		"no-console": "warn",
```

Comment:
Why did you change it to warn from error?
Don't we *want* no console to get people to use `print`, `warn` & `error`?

Reply:
This is the compiler's eslint, not the user's. In the compiler, using console is no recommended, but not really an error, and it's annoying while debugging.

Reply:
Oh, okay! Thanks for clarifying.